azure-functions-gradle-plugin/CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # Change Log
 All notable changes to the "Azure Function Plugin for Gradle" will be documented in this file.
 - [Change Log](#change-log)
+  - [1.6.0](#160)
   - [1.5.0](#150)
   - [1.4.0](#140)
   - [1.3.0](#130)
   - [1.2.0](#120)
   - [1.1.0](#110)
   - [1.0.0](#100)
 
+## 1.6.0
+- Support [proxy](https://github.com/microsoft/azure-gradle-plugins/wiki/Proxy)in azure function gradle plugin
```

Comment:
miss a space [proxy](https://github.com/microsoft/azure-gradle-plugins/wiki/Proxy)in